tests/contract-playground/src/MsgValueInLoop.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
```

Comment:
Can you please add `// GOOD` examples, written using the recommendation we provide?

Reply:
Of course !